source/_patterns/01-elements/headline/_headline.hbs

```diff
@@ -1 +0,0 @@
-<h{{ type }} class="elm-headline {{ styleModifier }}"{{#if id}} id="{{id}}"{{/if }}>{{ text }}</h{{ type }}>
```

Comment:
I'm still not yet convinced that we should remove the headline element in favour of the HTML standard HTML elements. We should probably still discuss this again, as our current approach of styling native HTML elements with DB UI Base doesn't interfere with having a headline element as well, and additionally is only optional (library users might still only use DB UI Bases variables), and/or the ShadowDOM, so any of these native HTML elements references wouldn't work.

Reply:
We've decided to keep at least the element for the moment.